lib/git_worktree.rb

```diff
@@ -186,7 +186,7 @@ def save_changes(message, owner = :local)
   def file_attributes(fname)
     walker = Rugged::Walker.new(@repo)
     walker.sorting(Rugged::SORT_DATE)
-    walker.push(@repo.ref(local_ref).target)
+    walker.push(@repo.ref(local_ref).target.oid)
```

Comment:
I think this line is the only one of concern, but I don't really think we can test until we have it on a nightly build of an appliance, so I think this is fine just to "roll the dice" on this one and make fixes later if need be.

That said, I think this basically means we are _**NOT**_ backporting it or trying to get it in the `kasparov` release, which I think is fine.  If we haven't split off that branch, I think we should hold off on merging this.

Reply:
Docs say it should work, tests pass, shouldn't be an issue. (Famous last words?)

No, definitely not backporting.